aviatrix/resource_aviatrix_transit_gateway_peering.go

```diff
@@ -180,40 +265,75 @@ func resourceAviatrixTransitGatewayPeeringRead(d *schema.ResourceData, meta inte
 func resourceAviatrixTransitGatewayPeeringUpdate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*goaviatrix.Client)
 
-	var gw1Cidrs []string
-	for _, cidr := range d.Get("gateway1_excluded_network_cidrs").([]interface{}) {
-		gw1Cidrs = append(gw1Cidrs, cidr.(string))
-	}
-	var gw2Cidrs []string
-	for _, cidr := range d.Get("gateway2_excluded_network_cidrs").([]interface{}) {
-		gw2Cidrs = append(gw2Cidrs, cidr.(string))
-	}
+	d.Partial(true)
```

Comment:
Do we really need Partial state mode? Remember Partial state is totally removed from Terraform SDK v2

Reply:
Will need a PR in future to remove all of them in one try.